app/views/apis/list.html

```diff
@@ -182,6 +182,9 @@
         </li>
         <li role="presentation" class="nav-item">
             <a href="#" ng-click="setActiveTab(2,$event)" role="tab" aria-controls="apis" class="nav-link" ng-class="{active:isTabActive(2)}">APIs</a>
+        </li>
```

Comment:
You cannot implement this like this because it means that there will be a statuscode tab no matter whether there is actually status code content or not.  Note that there is an if block above, <ul ng-if="overviewHtml" id="tabs" class="nav" role="tablist"> that covers the case that there is an overviewHtml or not.  You need to introduce the same sort of logic where if there is a statusCodeHtml, only then is the tab displayed.  Otherwise it is not displayed at all.  This maintains compatibility for other users of API Explorer.

Reply:
Added ng show and ng if